integration/pubsub_test.go

```diff
@@ -401,13 +471,26 @@ func checkResultHandler(publishedMsg string, publishedMsgType string, ch chan<-
 				return message.Error(errors.New("published message type and received message type are different"))
 			}
 			if publishedMsgType == reflection.GetType(retryLaterEvent{}) {
-				//use delivery count now that retry later abandons
+				// use delivery count now that retry later abandons
 				if msg.Message().DeliveryCount == 2 {
 					ch <- true
 					return message.Complete()
 				}
 				return message.RetryLater(1 * time.Second)
 			}
+			if publishedMsgType == reflection.GetType(shortLockMessage{}) {
+				// the shortlock is set at 1 second in the listener setup
+				// We sleep for longer to trigger the lock renewal in the listener.
+				// if the renewal works, then the message completion will succeed.
+				time.Sleep(3 * time.Second)
```

Comment:
crank this up for at least one test and see if we can see any of the renew failures  that @xhl873  saw?

Reply:
Will try. I think what @xhl873 saw was message expiration, which prevents any future actions on the message (TTL).
to be confirmed.